packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2867,4 +2870,38 @@ export default class Meeting extends StatelessWebexPlugin {
 
     return this.updateMedia({mediaSettings: this.mediaProperties.mediaDirection});
   }
+
+  /**
+   * Sets the quality level of all meeting media (incoming/outgoing)
+   * @param {String} level {LOW|MEDIUM|HIGH}
+   * @returns {Promise}
+   */
+  setMeetingQuality(level) {
+    LoggerProxy.logger.log(`Meeting:index#setMeetingQuality --> Setting quality to ${level}`);
+
+    if (!QUALITY_LEVELS[level]) {
+      const errorMessage = `Meeting:index#setMeetingQuality --> ${level} not defined`;
+
+      LoggerProxy.logger.error(errorMessage);
+
+      return Promise.reject(new Error(errorMessage));
+    }
+
+    // Set the overall quality level
+    this.mediaProperties.setQualityLevel(level);
+
+    const qualityActions = [];
+
+    const {receiveAudio, receiveVideo, sendVideo} = this.mediaProperties.mediaDirection;
```

Comment:
Can you add a comment on what this is doing, and how it's different from setting the media properties? Or explain the steps a bit more